kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/main/java/org.jbpm.notification.listeners/CustomNotificationListener.java

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Seems right package structure folder has not been created properly.

It should be `org/jbpm/notification/listeners/` but the actual folder is just `org.jbpm.notification.listeners`

Reply:
my bad, fixed